spec/index.html

```diff
@@ -167,57 +167,71 @@ <h3>Resources and Statements</h3>
       exists, without explicitly naming it.</p>
   </section>
 
-  <section id="section-quoted-triples">
-    <h3>Quoted Triples</h3>
+  <section id="section-graph-terms">
+    <h3>Graph Terms</h3>
 
-    <p>A <a>quoted triple</a> is an <a>RDF term</a> with the components of
-      an <a>RDF triple</a>, which can be used as the <a>subject</a> or <a>object</a>
+    <p>A <a>graph term</a> is an <a>RDF term</a> with the components of
+      an <a>RDF graph</a>, which can be used as the <a>subject</a> or <a>object</a>
       of another triple.</p>
 
-    <p>A <a>quoted triple</a> is not necessarily asserted, allowing
+    <p>A <a>graph term</a> is not necessarily asserted, allowing
       statements to be made about other statements that may not be
-      asserted within an <a>RDF graph</a>.
+      asserted within another <a>RDF graph</a>.
       This allows statements to be made about relationships that may be contradictory.
-      For a <a>quoted triple</a> to be asserted,
-      it must also appear in a graph as an <a>asserted triple</a>.</p>
+      For a <a>graph term</a> to be asserted,
+      the triples it contains must also appear in a graph which is not a <a>graph term</a>.</p>
 
-    <p>The following diagram represents a statement with an unasserted <a>quoted triple</a>
+    <p>The following diagram represents a statement with an unasserted <a>graph term</a>
```

Comment:
This sentence and the rest of the text about the examples/figures in this section needs to be revised (assuming that we won't have a notion of an unasserted graph term but rather unasserted triples in a graph term).

Reply:
The substance of the diagrams should be fine, although they should be renamed accordingly, and we should have a diagram or two that includes a graph term with more than one triple.

Reply:
```suggestion
    <p>The following diagram represents a statement with <a>graph term</a> containing unasserted triples
```